test/javascripts/unit/lib/ai-streamer-test.js

```diff
@@ -1,5 +1,53 @@
 import { module, test } from "qunit";
-import { addProgressDot } from "discourse/plugins/discourse-ai/discourse/lib/ai-streamer";
+import {
+  addProgressDot,
+  applyProgress,
+  MIN_LETTERS_PER_INTERVAL,
+} from "discourse/plugins/discourse-ai/discourse/lib/ai-streamer";
+
+class FakeStreamUpdater {
+  constructor() {
+    this._streaming = true;
```

Comment:
The preferred syntax for private vars is to use `#` now, e.g. `this.#streaming = true`

Reply:
You just need to define them at the top of the class like this:

https://github.com/discourse/discourse/blob/f5fc7790ce7128e99c32e14157506ff2636b239d/app/assets/javascripts/discourse/app/services/loading-slider.js#L15-L18

Reply:
oh will fix